SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -519,7 +519,7 @@
         WaferAngles      = cms.untracked.vdouble(90.0,30.0),
         WaferSize        = cms.untracked.double(123.7),
         MouseBite        = cms.untracked.double(2.5),
-        CheckID          = cms.untracked.bool(True),
```

Comment:
What is an implication on turning this CheckID to off?

Reply:
This parameter was not used earlier. Now it is used to do some debug testing which is expensive in CPU time. So by default, it is better to set it as False and when checks are needed set it to True. So this change has no real effect on standard production